java/build/bazel/tests/integration/WorkspaceDriver.java

```diff
@@ -26,14 +28,16 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
+import java.util.Properties;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 public class WorkspaceDriver {
 
-  private static Path tmp;
+  static Path tmp;
```

Comment:
why does this need to be public?

Reply:
Sorry, this package-private is a side effect of me working before the workspace was accessible in read-only.  I'll re-check everything anyway as I plan to rebase on the reformatted base.